common/fee/types/config.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import "math/big"
+
+type BumpConfig interface {
+	LimitMultiplier() float32
+	PriceMax() *big.Int
+	BumpPercent() uint16
+	BumpMin() *big.Int
+	TipCapDefault() *big.Int
+}
+
+type FixedPriceEstimatorConfig interface {
+	BumpThreshold() uint64
+	FeeCapDefault() *big.Int
+	LimitMultiplier() float32
+	PriceDefault() *big.Int
+	TipCapDefault() *big.Int
```

Comment:
For the dynamic static estimator - maybe you would have to use the generic fee calculation twice for the FeeCap and the TipCap?

not sure if this would actually work, I would need a closer look at the FeeCap and TipCap logic, but just tossing out an idea